### ISSUE TITLE
unable to build reanimated with latest react due to turbo module classes removal

### ISSUE THREAD

**Issue by Icehunter** [software-mansion/react-native-reanimated#5574]
### Description

React Native starting with the 0.74 nighties has removed the classes for TurboModules and made them internal.

https://github.com/facebook/react-native/pull/39682

With this change, I am unable to use the library with latest RN.

```
node_modules/react-native-reanimated/android/src/paper/java/com/swmansion/reanimated/NativeProxy.java:9: error: package com.facebook.react.turbomodule.core does not exist
import com.facebook.react.turbomodule.core.CallInvokerHolderImpl
```

### Steps to reproduce

1. setup any sample app with react-native@nightly along with latest reanimated
2. try to build

### Snack or a link to a repository

https://github.com/facebook/react-native/pull/39682

### Reanimated version

3.6.1

### React Native version

0.74-0 nightly

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

Fabric (New Architecture)

### Build type

Release app & production bundle

### Device

Real device

### Acknowledgements

Yes

**Comment by Icehunter** [comment]
Sorry just tried with react-native-reanimated@next and everything works as expected.

**Comment by Icehunter** [comment]
Wrong button sorry. It's still issue with @next as well.

**Comment by zahidhussain998** [comment]
yep still issue is but i'm trying to solve it

**Comment by piaskowyk** [comment]
Hey, we haven't been supporting RN@0.74 :/ We are working on it here - https://github.com/software-mansion/react-native-reanimated/pull/5328

**Comment by Rexogamer** [comment]
any news on RN 0.74 support? facing a different error and saw this; the linked PR was closed without merging and I can't seem to find anything else about 0.74

**Comment by Gu1K** [comment]
same

**Comment by Rexogamer** [comment]
[another pr](https://github.com/software-mansion/react-native-reanimated/pull/5815) was opened and merged, so it *should* just be a matter of cutting a new release

**Comment by Rexogamer** [comment]
fixed in v3.9.0-rc.0 :tada:
